public/static/docs/command-reference/checkout.md

```diff
@@ -25,7 +25,12 @@ workspace data with the versions specified in the current DVC-files.
 that changes the DVC-files in the workspace.
 
 💡 For convenience, a Git hook is available to automate running `dvc checkout`
-after `git checkout`. Use `dvc install` to install it.
+after `git checkout`. See the
+[**Automating** example](#example-automating-dvc-checkout) below or
+`dvc install` for more details.
+
+> Note that the hooks only have an effect in repository versions (branches,
```

Comment:
`Note that hook won't activate if the branch (commit, tag, etc) you checkout does not have DVC initialized (no .dvc directory)`?

Reply:
Will use suggestion for https://github.com/iterative/dvc.org/pull/931#pullrequestreview-348280129, removing this one...